### PR TITLE
Update port references in hokusai config

### PR DESCRIPTION
**Pull request by eessex**
Updates staging/production config to use named ports.

### CODE REVIEW

**Comment by izakp** [comment]
👍 on changing the port to 8080 but we shouldn't change it until _after_ we use named ports and deploy with Hokusai v0.5.6

**Comment by eessex** [comment]
Cool, just switched it back to 5000 for now

File: hokusai/production.yml [hunk 1]
```diff
@@ -118,11 +130,11 @@ spec:
     - port: 80
       protocol: TCP
       name: http
-      targetPort: 8080
+      targetPort: nginx-http
     - port: 443
       protocol: TCP
       name: https
-      targetPort: 8080
+      targetPort: nginx-https
```
Comment (eessex) [inline hokusai/production.yml]: Wanted to ask for a second look on this one, should they both point at `force-http` instead?

Comment (izakp) [reply hokusai/production.yml]: Yeah so the ELB is terminating SSL here so it should go to `nginx-http`

File: hokusai/production.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
----
+----
```
Comment (izakp) [inline hokusai/production.yml]: one extra `-`?

File: hokusai/staging.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+----
```
Comment (izakp) [inline hokusai/staging.yml]: Same - from https://yaml.org/spec/1.0/
> YAML uses three dashes (“---”) to separate documents within a stream.

Comment (eessex) [reply hokusai/staging.yml]: i just removed both of them, which is what my linter prefers

**Comment by izakp** [comment]
🎉 will merge and update

**Pull request merged**